app/code/Magento/ProductVideo/README.md

```diff
@@ -1 +1,44 @@
-The Magento_ProductVideo module implements functionality related with linking video files from external resources to product.
+# Magento_ProductVideo module
+
+This module implements functionality related with linking video files from external resources to product.
+
+## Installation
+
+Before installing this module, note that the Magento_ProductAlert is dependent on the following modules:
+- `Magento_Catalog`
+- `Magento_Backend`
+
+The Magento_ProductVideo module creates the `catalog_product_entity_media_gallery_value_video` table in the database.
```

Comment:
Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.